devolving_music/lib/song_scores.py

```diff
@@ -0,0 +1,117 @@
+from typing import Iterable
+from bisect import bisect_left
+
+
+from devolving_music.lib.elo_scoring import elo_rating
+from devolving_music.models.event import Event
+from devolving_music.models.song_submission import SongSubmission
+from devolving_music.models.song_comparison import SongComparison
+from devolving_music.lib.score_object import ScoreObject
+
+import pdb 
+#use for debugging
+# pdb.set_trace()
+
+class SongScores():
+    def __init__(self, event: Event):
+        # self.song_submissions = SongSubmission.get_voteable_submissions(Event)
+        self.song_submissions = SongSubmission.get_voteable_submissions_dict(event)
+        self.comparison_submissions = SongComparison.get_event_comparisons(event)
+
+    @staticmethod
+    def compare_logged(comparison_submission: "SongComparison", song1: "ScoreObject", song2: "ScoreObject"):
```

Comment:
just realized we reversed the polarity on this method name - it actually returns true if the comparison is *not* logged, so it should be named accordingly.